workfinder/search/s1.py

```diff
@@ -1,31 +1,51 @@
 import logging
 import math
+import os
+from pathlib import Path
 
 import numpy
 import pandas as pd
 from libcatapult.storage.s3_tools import S3Utils
 from sentinelsat import SentinelAPI
 
 from workfinder import get_config
-from workfinder.search import get_aoi
+from workfinder.search import get_aoi, get_redis_connection, list_s3_files, get_ard_list
 from workfinder.search.BaseWorkFinder import BaseWorkFinder
 
 
 class S1 (BaseWorkFinder):
 
+    def submit_tasks(self, to_do_list):
```

Comment:
the use of type hints is something that really helps to quickly understand what types are expected to be passed and be returned by a function